src/analysis/normed_space/conformal_linear_map.lean

```diff
@@ -59,6 +58,10 @@ lemma is_conformal_map_id : is_conformal_map (id R M) :=
 lemma is_conformal_map_const_smul {c : R} (hc : c ≠ 0) : is_conformal_map (c • (id R M)) :=
 ⟨c, hc, id, by ext; simp⟩
 
+lemma is_conformal_map_linear_isometry (f' : E →ₗᵢ[ℝ] F) :
```

Comment:
Can you please namespace this as `linear_isometry.is_conformal_map` to allow for projection notation?

Reply:
Done